oauth2client/contrib/django_orm.py

```diff
@@ -16,6 +16,12 @@
 
 Utilities for using OAuth 2.0 in conjunction with
 the Django datastore.
+
+Note that only supported versions of Django are supported
```

Comment:
This sentence reads a little funny, how about just "Only Django versions 1.8+ are supported."?